template/src/locale/I18nConfig.ts

```diff
@@ -1,30 +1,34 @@
 import i18n from 'i18next'
-import {initReactI18next} from 'react-i18next'
+import {getI18n, initReactI18next} from 'react-i18next'
 
 import en from './en'
 
-i18n.use(initReactI18next).init({
-  compatibilityJSON: 'v3', //https://stackoverflow.com/questions/70493788/i18nextpluralresolver-your-environment-seems-not-to-be-intl-api-compatible-u
-  lng: 'en',
-  fallbackLng: 'en',
+export const configureLocalization = (locale: string, fallback = 'en') => {
+  return i18n.use(initReactI18next).init({
+    returnNull: false,
+    lng: locale,
+    fallbackLng: fallback,
 
-  resources: {
-    en: {
-      translation: en,
+    resources: {
+      en: {
+        translation: en,
+      },
     },
-  },
 
-  debug: false,
+    debug: false,
 
-  cache: {
-    enabled: true,
-  },
+    cache: {
+      enabled: true,
+    },
 
-  interpolation: {
-    escapeValue: false, // not needed for react as it does escape per default to prevent xss!
-  },
-})
+    interpolation: {
+      escapeValue: false, // not needed for react as it does escape per default to prevent xss!
+    },
+  })
+}
 
-export const getString = (key: string, params?: any): string => i18n.t(key, params)
+export const getString = (key: keyof typeof en, params?: any): any => {
```

Comment:
Should the return type being `string` rather than `any`?

Reply:
string is not type of TFunctionDetailedResult which is define by getI18n(). and also getString must satisfy type ReactNode of <Text> component

Reply:
Okay then we will move this to other ticket